docs/pages/infrastructure/dns-and-domain-registration-security.mdx

```diff
@@ -0,0 +1,395 @@
+
+**Setup**: Add an SPF record to your DNS zone:
+```
+example.com. TXT "v=spf1 include:_spf.google.com ~all"
+```
+
+#### DKIM (DomainKeys Identified Mail)
+DKIM adds a digital signature to your emails, allowing receiving servers to verify that emails actually came from your domain and haven't been tampered with.
+
+**How it protects you**: DKIM signatures prove email authenticity and integrity, making it much harder for attackers to successfully spoof your domain in phishing campaigns.
+
+**Setup**: Configure with your email provider and publish public keys in DNS (your email provider will provide the specific records).
+
+#### MTA-STS (Mail Transfer Agent Strict Transport Security)
+MTA-STS enforces encrypted connections between mail servers, preventing man-in-the-middle attacks on email transmission.
+
+**How it protects you**: Without MTA-STS, emails can be intercepted in transit. This is especially critical for password reset emails and other sensitive communications.
+
+**Setup**: Add an MTA-STS record to your DNS zone:
+```
+_mta-sts.example.com. TXT "v=STSv1; id=20240101000000"
+```
+
+#### DMARC (Domain-based Message Authentication)
+DMARC builds on SPF and DKIM to provide policy enforcement for email authentication. It tells receiving mail servers what to do with emails that fail authentication checks.
+
+**How it protects you**: DMARC prevents email spoofing by instructing receiving servers to reject or quarantine emails that fail authentication, protecting your users from phishing attacks.
+
+**Setup**: Add a DMARC record to your DNS zone:
+```
+_dmarc.example.com. TXT "v=DMARC1; p=reject; rua=mailto:dmarc@example.com; ruf=mailto:forensics@example.com"
+```
+
+## Registrar Security
+
+### 1. Choosing a Secure Registrar
+Your domain registrar is the company that manages your domain registration with the central registry. This is often the weakest link in domain security, as many registrars have poor security practices and are vulnerable to social engineering attacks.
+
+#### Enterprise-Grade Registrars (Recommended)
+These registrars are designed for high-value domains and have security measures that consumer registrars lack:
+
+- **MarkMonitor**: Used by Fortune 500 companies, requires legal documentation for changes, dedicated security team
+- **AWS Route53**: IAM policy integration, CloudTrail logging, uses Amazon Registrar for major TLDs (but check TLD support)
+- **Cloudflare Registrar**: No markup pricing, automatic DNSSEC, built-in DDoS protection, requires Cloudflare services
+
+#### Consumer Registrars to Avoid for Critical Domains
+These registrars are designed for personal use and lack the security measures needed for Web3 projects:
+
+- **GoDaddy**: History of social engineering vulnerabilities, call center support vulnerable to manipulation
```

Comment:
There's information from ICANN showing the termination of several Registrars because of issues, there are even breaches that ended up in termination, super interesting: https://www.icann.org/compliance/notices

Example of godaddy:
https://threatpost.com/godaddy-employees-tricked-compromise-cryptocurrency/161520/
https://krebsonsecurity.com/2023/02/when-low-tech-hacks-cause-high-impact-breaches/